proposals/lip-0063.md

```diff
@@ -469,31 +474,32 @@ def createValidatorsArray():
     validators = []
     validatorKeys = getValidatorKeys()
     for every account in accounts:
-        if account.pos.validator.username == "":
+        if account.dpos.delegate.username == "":
             continue
         validator = {}
         validator.address = account.address
-        validator.name = account.pos.validator.username
+        validator.name = account.dpos.delegate.username
         validator.blsKey = INVALID_BLS_KEY
         validator.proofOfPossession = DUMMY_PROOF_OF_POSSESSION
         if validatorKeys[account.address]:
             validator.generatorKey = validatorKeys[account.address]
         else:
             validator.generatorKey = INVALID_ED25519_KEY
-        validator.lastGeneratedHeight = account.pos.validator.lastForgedHeight
```

Comment:
Note that the property name in LIP 57 is `lastGeneratedHeight`, see https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#genesis-assets-schema.

Reply:
I think this is a correct line:
`validator.lastGeneratedHeight = account.dpos.delegate.lastForgedHeight`
I think it is in the LIP. Let me know if you think something should be changed.

Reply:
All good, I guess I could confused which account is the v3 and which one is the v4.